### DISCUSSION TITLE
Adguard app(windows) vs Adguard home server

### DISCUSSION THREAD

**Discussion by scollar71** [AdguardTeam/AdGuardHome#4554] [General]
Can someone answer why it is that Adguard home server doesn't block as much as Adguard windows application, I assume it has to with blocklists however I have been unable to implement the correct blocklist(or whitelist) set up to mimic the app? Can anyone suggest a set that would be comparable?

**Comment by EugeneOne1** [comment]
@scollar71, AdGuardHome is the DNS-based advertisment blocker so it only filters DNS requests.  The application also filters the traffic itself.  It's recommended to use both since AGH makes it possible to avoid loading redundant data while the application processes the advertisements which couldn't be caught with DNS.

**Comment by scollar71** [comment]
I have seen that suggestion however when using both it appears after an uncertain amount of time I loose DNS connectivity. Even using bare minimal list, Any thoughts on this? And even before losing DNS connectivity page load times are VERY slow

**Reply by joshqou** [reply]
Are you still able to load the AdGuard Home dashboard once this happens?
